docs/src/pages/start/quasar-cli.md

```diff
@@ -24,6 +24,11 @@ $ yarn create quasar
 
 $ npm i -g @quasar/cli
 $ npm init quasar
+
```

Comment:
There is now a tip section called "Requirements" above this. It says "Yarn v1 (strongly recommended) or NPM."

![image](https://user-images.githubusercontent.com/6266078/172070791-e87df6f6-b928-43c1-889f-c1ad84df7c4c.png)

Please also update it accordingly.

Reply:
I ended up adding this myself into this PR.